src/java/org/apache/cassandra/schema/SchemaUpdateHandler.java

```diff
@@ -63,14 +64,15 @@
      * refreshed, the callbacks provided in the factory method are executed, and the updated schema version is announced.
      *
      * @param local whether we should reset with locally stored schema or fetch the schema from other nodes
-     * @return transformation result
      */
-    SchemaTransformationResult reset(boolean local);
+    void reset(boolean local);
 
     /**
      * Clears the locally stored schema entirely. After this operation the schema is equal to {@link DistributedSchema#EMPTY}.
      * The method does not execute any callback. It is indended to reinitialize the schema later using the method
      * {@link #reset(boolean)}.
+     *
+     * @return
```

Comment:
I think we should either add some description to the `@return` tag or remove it.

Reply:
ok, and btw. I suppose the more adequate names for those methods now would be reset -> sync, clear -> refresh